testproject/testapp/tests.py

```diff
@@ -616,3 +625,73 @@ def test_put_should_update_user(self):
         self.assert_status_equal(response, status.HTTP_200_OK)
         user = utils.refresh(user)
         self.assertEqual(data['email'], user.email)
+
+
+class SerializersManagerTest(SimpleTestCase):
+
+    def test_serializer_manager_init(self):
+        serializers_manager = djoser.serializers.SerializersManager({})
+        self.assertFalse(serializers_manager.serializers)
+
+    def test_get_serializer(self):
+        serializers_manager = djoser.serializers.SerializersManager({
+            'user': djoser.serializers.UserSerializer})
+        serializer_class = serializers_manager.get('user')
+        self.assertTrue(issubclass(serializer_class, djoser.serializers.UserSerializer))
+
+    def test_get_serializer_non_proper_name(self):
+        serializers_manager = djoser.serializers.SerializersManager({
+            'user': djoser.serializers.UserSerializer
+        })
+        self.assertRaises(Exception, serializers_manager.get, 'bad_name')
+
+    def test_get(self):
+        serializers_manager = djoser.serializers.SerializersManager({
+            'user': djoser.serializers.UserSerializer})
+        serializer_class = serializers_manager.get('user')
+        self.assertTrue(issubclass(serializer_class, djoser.serializers.UserSerializer))
+
+    def test_get_from_cache(self):
+        serializers_manager = djoser.serializers.SerializersManager({
+            'user': djoser.serializers.UserSerializer})
+        serializer_class = serializers_manager.get('user')
+        self.assertTrue(issubclass(serializer_class, djoser.serializers.UserSerializer))
+
+        with mock.patch.object(
+                djoser.serializers.SerializersManager, 'load_serializer') as load_serializer_mock:
+            serializer_class = serializers_manager.get('user')
+            self.assertTrue(issubclass(serializer_class, djoser.serializers.UserSerializer))
+            self.assertFalse(load_serializer_mock.called)
+
+    def test_get_serializer_class_not_exists(self):
+        serializers_manager = djoser.serializers.SerializersManager({})
+
+        if django.VERSION >= (1, 7):
+            self.assertRaises(ImportError, serializers_manager.load_serializer, 'bad.UserSerializer')
+        else:
+            from django.core.exceptions import ImproperlyConfigured
+            self.assertRaises(ImproperlyConfigured, serializers_manager.load_serializer, 'bad.UserSerializer')
```

Comment:
In fact, this test case shows that we can't load not existing class. This is evident in all. We do not support this exception in any way. So we don't need this test.